src/clj/rems/db/applications.clj

```diff
@@ -591,10 +616,10 @@
       :licenses licenses
       :phases (get-application-phases (:state application))})))
 
-(defn create-new-draft [wfid]
-  (let [uid (get-user-id)
-        id (:id (db/create-application! {:user uid :wfid wfid}))]
-    id))
+(defn create-new-draft [wfid user-id]
+  (assert wfid)
+  (assert user-id)
+  (:id (db/create-application! {:user user-id :wfid wfid})))
 
 (defn create-new-draft-at-time [wfid time]
   (let [uid (get-user-id)
```

Comment:
Are there any places where get-user-id is needed instead of getx-user-id? We could do a global search and replace to use getx-user-id everywhere.